win32/tests/gballoc_hl_perf/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#Copyright (c) Microsoft. All rights reserved.
+
+cmake_minimum_required(VERSION 2.8.11)

```

Comment:
Also think that perf tests should go in interfaces/tests (under if WIN32 for now until we have it working for Linux)

Reply:
Moved

---
In reply to: [459666667](https://github.com/Azure/azure-c-pal/pull/14#discussion_r459666667) [](ancestors = 459666667)